### PR TITLE
Use niv, revive gzseek, change nixpkgs pin

### PR DESCRIPTION
**Pull request by yorickvP**
Binwalk broke, it changed behaviour slightly. I removed the glib dependency from gzseek and inlined it in this repo. I also added niv.

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -163,18 +157,14 @@ let
...
 
-    HOME = ".";
-
-    nativeBuildInputs = [ binwalk ];
+    nativeBuildInputs = [ gzseek ];
 
-    unpackPhase = ''
-      binwalk --include compress --extract $src
-    '';
+    unpackPhase = "true";
```
Comment (mkaito) [inline default.nix]: Why true and not just empty?

File: README.md [hunk 2]
```diff
@@ -74,11 +74,12 @@ corresponds to a `deps/` path. Two types of deps are supported, `:git` and
 `:git` fetches given rev from `HEAD`. It doesn't support other branches just
 yet, see [NixOS/nix#2409](https://github.com/NixOS/nix/pull/2409).
 
-`:hexpm` fetches package from [Hex.pm](https://hex.pm). It comes with a SHA-256
-hash of catenation of Mix archive version, package metadata, and `.tar.gz`
-source code archive. Nix can only carry that catenation into the sandbox, so
-[`binwalk`](https://github.com/ReFirmLabs/binwalk) finds where the source code
-archive starts and extracts that part, ignoring archive version and metadata.
+`:hexpm` fetches package from [Hex.pm](https://hex.pm). It comes with
+a SHA-256 hash of catenation of Mix archive version, package metadata,
```
Comment (mkaito) [inline README.md]: While "catenation" is actually a real word, I had to look it up to confirm it wasn't a typo. Given that this is the same thing as *con*catenation, unless we're talking chemistry, maybe let's use less obscure words.

File: default.nix (hunk 1 above)
Comment (yorickvP) [reply default.nix]: Didn't work

File: README.md (hunk 2 above)
Comment (yorickvP) [reply README.md]:
```suggestion
a SHA-256 hash of concatenation of Mix archive version, package metadata,
```

Comment (mkaito) [reply README.md]: Don't forget other instances thereof

**Pull request merged**